mfr/extensions/office365/templates/viewer.mako

```diff
@@ -0,0 +1,11 @@
+<style>
+iframe {
+    width: 100%;
+    height: 800;
+}
+</style>
+
+<iframe src=${url} frameborder='0'></iframe>
```

Comment:
I get your point but not sure how 302 works. OSF creates the parent iframe and we shouldn't modify it from MFR. Is embedded iframe a bad practice? For MFR itself, it should use iframe here since it loads content from an untrusted external web service.